2D3D_UnityProject/Assets/Models/Puzzle Models/Koi_Puzzle/KoiBoi_Anim_CTRL.controller

```diff
@@ -0,0 +1,12 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!91 &9100000
+AnimatorController:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: Koi_Boi_
+  serializedVersion: 5
+  m_AnimatorParameters: []
+  m_AnimatorLayers: []
```

Comment:
where's all the stuff!!!

(i'll re-implement them myself since you said it's easy, don't worry)